tools/GraphClust/GSPAN/structure_to_gspan.xml

```diff
@@ -0,0 +1,111 @@
+        mkdir ./temp/ ./GSPAN_Outputs/ &&
+        structure_to_gspan.pl
+            --input-file '$dataFile'
+            --input-format 'rnafold'
+            $abstr
+            $stack
+            $seq_graph_t
+            -group $group
+            -tmp ./temp
+            -o ./GSPAN_Outputs/
+    ]]>
+    </command>
+    <inputs>
+        <param type="data" name="dataFile" format="data" label="Sequence and Structure input file in the prediction tool format" />
+        <param argument="-stack" truevalue="-stack" falsevalue="" checked="true"  type="boolean"
+            label="Add stacking information to graphs"/>
+        <param argument="abstr" truevalue="-abstr" falsevalue="" type="boolean"
+            label="Add abstract structure graphs to the single shrep graph instances."/>
+        <param argument="-seq-graph-t" name="seq_graph_t" truevalue="-seq-graph-t" falsevalue="" checked="true" type="boolean"
+            label="Add for each 't #' a graph which contains no structure"/>
+        <param name="group" type="integer" value="10000"
+            label="Group size" help="Default is 10000"/>
+    </inputs>
+    <outputs>
+        <collection name="gspan_compressed" type="list" label="GSPAN Groups" >
+            <discover_datasets pattern="(?P&lt;name&gt;.*)\.bz2$" directory="GSPAN_Outputs" />
+        </collection>
+    </outputs>
+    <tests>
+        <test>
+            <param name="dataFile" value="sample_3.struct"/>
+            <param name="i_stacks" value="True" />
+            <param name="i_abstr" value="False" />
+            <param name="seq_graph_t" value="True" />
+            <param name="group" value="10000" />
+            <output_collection name="gspan_compressed" type="list">
+                <element name="1.group.gspan" file="sample_3.1.group.gspan.bz2"/>
+            </output_collection>
+        </test>
+    </tests>
+    <help>
+    <![CDATA[
+
+**What it does**
+For each pair of sequence and structure provided in the input file
+and convert them into GSPAN format graphs.
+
+
+**Parameters**
+
+                HELP
+        -help   brief help message
+        -man    full documentation
+
+        COMPULSORY
+        -input-file     <STRING> e.g. "sequence.struct"
+                        The sequence and structure data, the output of structure prediction tool.
+
+        OPTIONS
+        -input-format Sequence Structure format of the input. Allowed format: "rnafold"
+        -stack          Adds stacking information to graphs. This adds an additional
+                        vertex (type P) for each pair of stacked base-pairs and four edges
+                        (type p) from each of the involved bases to the new vertex.
+        -seq-graph-win  add for each window a graph which contains no structure
+        -seq-graph-t    add for each 't #' a graph which contains no structure
+        -seq-graph-alph change the alphabet of unstructured graphs
+        -annotate               <STRING> annotation.tab
```

Comment:
Can you improve the help here? remove at least the options that are not offered? A short example of the expected input file would also be nice.